nuc_morph_analysis/lib/preprocessing/add_features.py

```diff
@@ -468,6 +614,26 @@ def sum_mitotic_events_along_full_track(df0, feature_list=[]):
 
     return sum_events_along_full_track(df0, feature_list)
 
+def normalize_sum_events(df_full, event_cols):
```

Comment:
suggestion: change to `event_cols = ['sum_has_mitotic_neighbor', 'sum_has_dying_neighbor']`, so it defaults to those columns

Reply:
Sure